app/javascript/components/Profile/FollowForm.tsx

```diff
@@ -74,9 +74,9 @@ export const FollowForm = ({
 };
 
 export const FollowFormBlock = ({ creatorProfile }: { creatorProfile: CreatorProfile }) => (
-  <div style={{ display: "grid", gap: "var(--spacer-8)", alignContent: "center", flexGrow: 1 }}>
+  <div className="flex grow flex-col justify-center gap-16 px-4 lg:px-0">
```

Comment:
```suggestion
  <div className="flex grow flex-col justify-center gap-16 px-4">
```

@abhix4 It doesn't seem to need this? (the design system styles override it)

Reply:
Ah I was looking at one of the other many variations of the profile layout ><

Makes sense then!